client/DAW/components/track-list.js

```diff
@@ -26,6 +26,7 @@ const TrackList = (props) => {
   const zoom = 200; // pixels per second
   return (
     <div className="track-list" style={styles.trackList(getWidth(clips, zoom))}>
+      <WaveformGradient />
```

Comment:
The SVG gradient actually gets mounted here, but it can be referenced remotely by all the other SVG on the page, using _WIZARDRY_